src/methods/defining_methods/hard/q3/Polygo.java

```diff
@@ -0,0 +1,29 @@
+package methods.defining_methods.hard.q3;
+
+/**
+ * [J] Write a Java Program that asks the user for the number of sides on a polygon, then determines the sum of the measure
```

Comment:
Change question text to: 
`Write a Java Program that asks the user for the number of sides on a polygon, then determines the sum of the measure of the interior angles of that polygon. Remember that the formula to determine the sum of interior angles is (n-2)*180, where n is the number of sides. Create a method called "innerAngles(int size)" that takes the number of sides and returns an integer as the sum of the interior angles. The input is guaranteed to be a positive integer greater or equal to 3.`